networkx/algorithms/centrality/eigenvector.py

```diff
@@ -11,42 +11,64 @@
 def eigenvector_centrality(G, max_iter=100, tol=1.0e-6, nstart=None, weight=None):
     r"""Compute the eigenvector centrality for the graph `G`.
 
-    Eigenvector centrality computes the centrality for a node based on the
-    centrality of its neighbors. The eigenvector centrality for node $i$ is
-    the $i$-th element of the vector $x$ defined by the equation
+    Eigenvector centrality computes the centrality for a node by adding
+    the centrality of its predecessors. The centrality for node $i$ is the
+    $i$-th element of a left eigenvector associated with the positive
+    eigenvalue $\lambda$ of maximum modulus. Such an eigenvector $x$ is
+    defined up to a multiplicative constant by the equation
 
     .. math::
 
-        Ax = \lambda x
+         \lambda x^T = x^T A,
+
+    where $A$ is the adjacency matrix of the graph `G`. By definition of
+    row-column product, the equation above is equivalent to
+
+    .. math::
+
+        \lambda x_i = \sum_{j\to i}x_j.
 
-    where $A$ is the adjacency matrix of the graph `G` with eigenvalue
-    $\lambda$. By virtue of the Perron–Frobenius theorem, there is a unique
-    solution $x$, all of whose entries are positive, if $\lambda$ is the
-    largest eigenvalue of the adjacency matrix $A$ ([2]_).
+    That is, adding the eigenvector centralities of the predecessors of
+    $i$ one obtains the eigenvector centrality of $i$ multiplied by
+    $\lambda$.
+
+    By virtue of the Perron–Frobenius theorem [1]_, if `G` is strongly
+    connected there is a unique such eigenvector $x$, and all its entries
```

Comment:
```suggestion
    connected there is a unique eigenvector $x$, and all its entries
```
I think this reads a little better, but it's just a suggestion - feel free to ignore!